pkg/controller/direct/metastore/mapper.generated.go

```diff
@@ -104,5 +294,266 @@ func MetastoreFederationSpec_ToProto(mapCtx *direct.MapContext, in *krm.Metastor
+	if in == nil {
+		return nil
+	}
+	out := &krm.NetworkConfig_ConsumerObservedState{}
+	// MISSING: Subnetwork
+	out.EndpointURI = direct.LazyPtr(in.GetEndpointUri())
+	out.EndpointLocation = direct.LazyPtr(in.GetEndpointLocation())
+	return out
+}
+func NetworkConfig_ConsumerObservedState_ToProto(mapCtx *direct.MapContext, in *krm.NetworkConfig_ConsumerObservedState) *pb.NetworkConfig_Consumer {
+	if in == nil {
+		return nil
+	}
+	out := &pb.NetworkConfig_Consumer{}
+	// MISSING: Subnetwork
+	out.EndpointUri = direct.ValueOf(in.EndpointURI)
+	out.EndpointLocation = direct.ValueOf(in.EndpointLocation)
+	return out
+}
+func Restore_FromProto(mapCtx *direct.MapContext, in *pb.Restore) *krm.Restore {
+	if in == nil {
+		return nil
+	}
+	out := &krm.Restore{}
+	// MISSING: StartTime
+	// MISSING: EndTime
+	// MISSING: State
+	// MISSING: Backup
+	// MISSING: Type
+	// MISSING: Details
+	return out
+}
+func Restore_ToProto(mapCtx *direct.MapContext, in *krm.Restore) *pb.Restore {
+	if in == nil {
+		return nil
+	}
+	out := &pb.Restore{}
+	// MISSING: StartTime
+	// MISSING: EndTime
+	// MISSING: State
+	// MISSING: Backup
+	// MISSING: Type
+	// MISSING: Details
+	return out
+}
+func RestoreObservedState_FromProto(mapCtx *direct.MapContext, in *pb.Restore) *krm.RestoreObservedState {
+	if in == nil {
+		return nil
+	}
+	out := &krm.RestoreObservedState{}
+	out.StartTime = direct.StringTimestamp_FromProto(mapCtx, in.GetStartTime())
+	out.EndTime = direct.StringTimestamp_FromProto(mapCtx, in.GetEndTime())
+	out.State = direct.Enum_FromProto(mapCtx, in.GetState())
+	out.Backup = direct.LazyPtr(in.GetBackup())
+	out.Type = direct.Enum_FromProto(mapCtx, in.GetType())
+	out.Details = direct.LazyPtr(in.GetDetails())
+	return out
+}
+func RestoreObservedState_ToProto(mapCtx *direct.MapContext, in *krm.RestoreObservedState) *pb.Restore {
+	if in == nil {
+		return nil
+	}
+	out := &pb.Restore{}
+	out.StartTime = direct.StringTimestamp_ToProto(mapCtx, in.StartTime)
+	out.EndTime = direct.StringTimestamp_ToProto(mapCtx, in.EndTime)
+	out.State = direct.Enum_ToProto[pb.Restore_State](mapCtx, in.State)
+	out.Backup = direct.ValueOf(in.Backup)
+	out.Type = direct.Enum_ToProto[pb.Restore_RestoreType](mapCtx, in.Type)
+	out.Details = direct.ValueOf(in.Details)
+	return out
+}
+func ScalingConfig_FromProto(mapCtx *direct.MapContext, in *pb.ScalingConfig) *krm.ScalingConfig {
+	if in == nil {
+		return nil
+	}
+	out := &krm.ScalingConfig{}
+	out.InstanceSize = direct.Enum_FromProto(mapCtx, in.GetInstanceSize())
+	out.ScalingFactor = direct.LazyPtr(in.GetScalingFactor())
+	return out
+}
+func ScalingConfig_ToProto(mapCtx *direct.MapContext, in *krm.ScalingConfig) *pb.ScalingConfig {
+	if in == nil {
+		return nil
+	}
+	out := &pb.ScalingConfig{}
+	if oneof := ScalingConfig_InstanceSize_ToProto(mapCtx, in.InstanceSize); oneof != nil {
```

Comment:
@jingyih @yuwenma @anhdle-sso - How do i fix this line. Its a enum. Need to conver in.InstanceSize (string) to enum which is a oneof definition.